aten/src/ATen/native/cuda/Gesv.cu

```diff
@@ -138,6 +160,35 @@ std::tuple<Tensor,Tensor> _gesv_helper_cuda(const Tensor& self, const Tensor& A)
   return std::tuple<Tensor,Tensor>(b_working_copy, A_working_copy);
 }
 
+std::tuple<Tensor&,Tensor&> _gesv_single_out_cuda(Tensor& result0, Tensor& result1,
+    const Tensor& self, const Tensor& A) {
+#ifndef USE_MAGMA
+AT_ERROR("gesv: MAGMA library not found in "
+    "compilation. Please rebuild with MAGMA.");
+#else
+  int64_t bx = self.size(0);
+  int64_t by = (self.dim() == 1) ? 1 : self.size(1);
```

Comment:
Could you update the docs on this? The docs currently say that `gesv` only supports 2D self (B variable).

It looks like the old THCTensor_(gesv) doesn't support 1D self but the THTensor_(gesv) supports both 2D and 1D self:
https://github.com/pytorch/pytorch/blob/7b375ed362c4da833c38691b6bf3ade0941d3bf1/aten/src/THC/generic/THCTensorMathMagma.cu#L66-L67